sequel-ace.xcodeproj/project.pbxproj

```diff
@@ -123,6 +123,7 @@
 		1A4CB03425923C4B00EDF804 /* StringRegexExtension.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1A4CB03325923C4B00EDF804 /* StringRegexExtension.swift */; };
 		1A4CB04F2592535300EDF804 /* StringRegexExtension.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1A4CB03325923C4B00EDF804 /* StringRegexExtension.swift */; };
 		1A4CB06B25926D7C00EDF804 /* StringRegexExtension.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1A4CB03325923C4B00EDF804 /* StringRegexExtension.swift */; };
+		1A4CE30D25B976AA00A6795C /* naughty_strings.txt in Resources */ = {isa = PBXBuildFile; fileRef = 1A4CE30C25B976AA00A6795C /* naughty_strings.txt */; };
```

Comment:
Just making sure, are we copying this file to released resources? Because we should only need it for testing, perhaps we could selectively copy it only if building for testing?

Reply:
Just for testing, I thought I added to just the unit test target, will double check.